### ISSUE TITLE
Bug: Calendar opens non-centered and unreadable.

### ISSUE THREAD

**Issue by namtrah** [namikaze-40p/obsidian-date-inserter#7]
On my screen, when I try to open the calendar, it is not centered in the window you open. In fact it is not possible to scroll it enough to the left to see the entire calendar.
![image](https://github.com/user-attachments/assets/f76b609d-b91c-448c-bc55-8a091eee4f28)

And when I try to scroll:
![image](https://github.com/user-attachments/assets/97f8a2bc-4178-4c71-be4b-25b6614d748b)

**Comment by namikaze-40p** [comment]
Perhaps the cause is that the "Date Inserter" css style is being overwritten by one of the following.
However, since I can't know your environment, I can't know the direct cause.

- [Themes](https://help.obsidian.md/Extending+Obsidian/Themes)
- [CSS snippets](https://help.obsidian.md/Extending+Obsidian/CSS+snippets)
- [Community plugins](https://help.obsidian.md/Extending+Obsidian/Community+plugins)

If possible, disabling all of them and then repeating the process of checking the "Date Inserter" while enabling them one at a time may help determine the cause.
If you can identify the cause, I can build the same environment and suggest a more specific solution.

---
Or since I have looked at your situation and thought of the css you need, could you please create and apply the following [CSS snippets](https://help.obsidian.md/Extending+Obsidian/CSS+snippets)?

```css
.modal.date-inserter-modal .datepicker {
  right: initial !important;
}
```

**Comment by namtrah** [comment]
Hi, I have confirmed that this happens when the plugin Cardboard. Has been installed. Have you tested with this?

P.s. the fix you gave above does not work, when Cardboard is installed.

**Comment by namtrah** [comment]
https://github.com/roovo/obsidian-card-board is the plugin in question

**Comment by namikaze-40p** [comment]
Thanks for your reply!
I was able to reproduce the same situation by installing "Cardboard".

I released [v0.3.4](https://github.com/namikaze-40p/obsidian-date-inserter/releases/tag/0.3.4) that resolves it.
Try it if you like.

**Comment by namtrah** [comment]
Confirmed this fixes the problem. Big THANK YOU!
